src/components/Dashboard/TodoList.vue

```diff
@@ -191,7 +207,7 @@
                 <v-btn v-else-if="!isCompleted(item)"
                   class="btn-file-now"
                   color="primary"
-                  :disabled="!item.enabled || coaPending || !confirmCheckbox"
+                  :disabled="!item.enabled || coaPending || !confirmCheckbox || hasPendingCorrection"
```

Comment:
We should reuse the `hasBlockerFiling` functionality instead of adding extra flags. Is this possible here?

Reply:
I'll look into it, everything is possible :)

Reply:
Done 👍